scripts/_docker.sh

```diff
@@ -76,6 +78,65 @@ function _docker_build_with_custom_tag() {
     cd $root
 }
 
+function _docker_update_service() {
+    local account=$1
+    local env=$2
+    local service=$3
+
+    if [[ -z ${account} ]]; then
+        echo "Account is required" >&2
+        return 1
+    fi
+
+    if [[ -z ${env} ]]; then
+        echo "Env is required" >&2
+        return 1
+    fi
+
+    if [[ -z ${service} ]]; then
+        echo "Service (back-end|ingestion|front-end) is required" >&2
+        return 1
+    fi
+
```

Comment:
Might be good to add some output here:-
```suggestion
echo "Starting Docker update for service '${service}' in environment '${env}' under account '${account}'..."
```

Reply:
If anything I think we have enough logs it's getting noisy so I'm gonna omit this